### PR TITLE
Add a "why not contribute?" banner to documentation

### PR DESCRIPTION
**Pull request by tn12787**
addresses #671.

### CODE REVIEW

**Comment by tn12787** [comment]
preview: https://www.cyberbotics.com/doc/reference/index?version=enhancement-contrib-banner

**Comment by tn12787** [comment]
👍

File: css/webots-doc.css [hunk 1]
```diff
@@ -357,6 +357,44 @@
...
+  font-family: "Roboto";
+  border-bottom: 1px solid #e2e2e2;
+  z-index: 1;
+  transform: translateY(-250%);
+  transition-property: transform;
+  transition-duration: 0.8s;
+  transition-timing-function: ease-out;
+}
+
+.webots-doc .visible {
```
Comment (fabienrohrer) [inline css/webots-doc.css]: Isn't this class name to generic?

**Comment by tn12787** [comment]
👍

**Pull request merged**